internal/spokes/spokes.go

```diff
@@ -578,14 +621,21 @@ func startSidebandMultiplexer(stderr io.ReadCloser, output io.Writer, capabiliti
 func (r *SpokesReceivePack) getAlternateObjectDirsEnv() []string {
 	return []string{
 		fmt.Sprintf("GIT_OBJECT_DIRECTORY=%s", r.quarantineFolder),
-		fmt.Sprintf("GIT_ALTERNATE_OBJECT_DIRECTORIES=%s", filepath.Join(r.repoPath, "objects")),
+		fmt.Sprintf("GIT_ALTERNATE_OBJECT_DIRECTORIES=%s:%s", filepath.Join(r.repoPath, "objects"), filepath.Join(r.repoPath, ".git/objects")),
```

Comment:
Why is `.git/objects` also included as an alternate?

Reply:
In scenarios where the remote wasn't a bare repo it was failing, so we had to add it to the objects could be found (since we are changing the GIT_OBJECT_DIRECTORY and setting it to `quarantineFolder`).